cli.sh

```diff
@@ -1163,22 +1167,26 @@ list_shared_hook_repos() {
                 unset IFS
 
                 set_shared_root "$LIST_ITEM"
-                if [ -d "$SHARED_ROOT/.git" ]; then
-                    if [ "$(git -C "$SHARED_ROOT" config --get remote.origin.url)" = "$LIST_ITEM" ]; then
+
+                if [ "$SHARED_REPO_IS_CLONED" = "true" ]; then
+                    if [ -d "$SHARED_ROOT/.git" ]; then
+                        if [ "$(git -C "$SHARED_ROOT" config --get remote.origin.url)" = "$SHARED_REPO_CLONE_URL" ]; then
+                            LIST_ITEM_STATE="active"
+                        else
+                            LIST_ITEM_STATE="invalid"
+                        fi
+                    else
+                        LIST_ITEM_STATE="pending"
+                    fi
+                else
+                    if [ -d "$SHARED_ROOT/.git" ]; then
                         LIST_ITEM_STATE="active"
                     else
                         LIST_ITEM_STATE="invalid"
                     fi
-                else
-                    LIST_ITEM_STATE="pending"
                 fi
 
-                if [ -n "$LIST_WITH_URL" ]; then
```

Comment:
Why did you remove this here?
Since we're normalizing the name, I think it might be good if we could continue to print out the actual source of where these came from.
What do you think?

Reply:
I do that be default now. Before we showed the weird mangled name (NORMALIZED_NAME, removed this) , which is any way a bit strange.
So because the user does not need to know this, I print the URL/path he entered in the list, which is also directly what we used to clone, that all these repos are in `.githooks/shared/<checksum>-40charsAbreviation` is internal stuf...